pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -74,7 +75,8 @@ type BuildSpec struct {
 
 	// Time after which the build times out. Defaults to 10 minutes.
 	//  Go's ParseDuration documentation for expected format: https://golang.org/pkg/time/#ParseDuration
-	Timeout string `json:"timeout,omitempty"`
+	//Timeout string `json:"timeout,omitempty"`
```

Comment:
Delete this commented-out line, and now you can probably also remove the line about Go's ParseDuration format.